config/wafv2/config.go

```diff
@@ -0,0 +1,18 @@
+package wafv2
+
+import "github.com/upbound/upjet/pkg/config"
+
+// Configure adds configurations for wafv2 group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_wafv2_web_acl_association", func(r *config.Resource) {
+		r.References = config.References{
+			"resource_arn": config.Reference{
```

Comment:
[Looks like](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl_association#resource_arn) `resource_arn` may refer to ARNs of multiple kinds of resources (Application Load Balancer, an Amazon API Gateway stage, or an Amazon Cognito User Pool). So, we should not configure a referencer here.

Reply:
@ulucinar same question here: https://github.com/upbound/provider-aws/pull/478#discussion_r1223018144, can we still configure the reference resolver to the most common resource?